nstack-translations-generator/Classes/TranslationsGenerator/Generator.swift

```diff
@@ -115,14 +117,16 @@ struct Generator {
         return modelString
     }
 
-    fileprivate static func generateSubModelsFromParserOutput(_ output: ParserOutput) throws -> (models: String, extensions: String) {
+    fileprivate static func generateSubModelsFromParserOutput(_ output: ParserOutput, _ settings: GeneratorSettings) throws -> (models: String, extensions: String) {
         var modelsString = ""
         var extensionsString = ""
 
         var indent = Indentation(level: 1)
 
         for case let (key, value as [String: AnyObject]) in output.language {
-            var subString = "\n\n" + indent.string() + "public final class \(key.uppercasedFirstLetter) {\n"
+            var subString = ""
+            settings.availableFromObjC ?    ( subString = "\n\n" + indent.string() + "@objc public final class \(key.uppercasedFirstLetter) : NSObject {\n" ) :
+                                            ( subString = "\n\n" + indent.string() + "public final class \(key.uppercasedFirstLetter) {\n" )
```

Comment:
I think a bit easier way of writing this would be:

~~~
let prefix = (settings.availableFromObjC ? "@objc " : "") + "public final class "
var subString = "\n\n" + indent.string() + prefix + "\(key.uppercasedFirstLetter) {\n"
~~~